src/main/kotlin/app/revanced/patches/twitter/misc/predictiveback/patch/PredictiveBackPatch.kt

```diff
@@ -0,0 +1,30 @@
+package app.revanced.patches.twitter.misc.predictiveback.patch
+
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.ResourceContext
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.ResourcePatch
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patches.twitter.misc.predictiveback.annotations.PredictiveBackCompatibility
+
+@Patch
+@Name("predictive-back")
+@Description("Enables the new predictive back feature")
+@PredictiveBackCompatibility
```

Comment:
Does this patch technically not work with every app? If so, you can remove this constraining annotation. The patch would also require to check if the attribute already exists.

Reply:
Yes, it does. But in which folder should I move it into?